### PR TITLE
Adding personal keymap

### PR DESCRIPTION
**Pull request by cankoklu**
## Description
Personal keymap with specific shortcuts etc.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/splitkb/kyria/keymaps/cankoklu/config.h [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#ifdef RGBLIGHT_ENABLE
+#    define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/cankoklu/config.h:20]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
// #define RGBLIGHT_EFFECT_RGB_TEST
// #define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+    _QWERTY = 0,
+    _NAV,
+	_SEL,
+	_NUM,
+    _SYM,
+    _FUNCTION,
+};
+
+enum custom_keycodes {
+  LINKIFY,
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c:29]:
```suggestion
  LINKIFY = SAFE_RANGE,
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by daskygit — COMMENTED** [review]
You don't need to `break;` as you're already using `return false;` above them, they're never reached.

File: keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+      return false;
+      break;	
+	  	  
+      case LINKIFY:
+      if (record->event.pressed)
+      {
+        SEND_STRING(SS_LGUI("k") SS_LGUI("v") SS_TAP(X_ENTER));
+      }
+      return false;
+      break;
```
Comment (daskygit) [inline keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c:306]:
```suggestion
```

File: keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+ 
+      case THUMBUP:
+      if (record->event.pressed)
+      {
+        SEND_STRING("1t1" SS_TAP(X_DOWN) SS_TAP(X_ENTER));
+      }
+      return false;
+      break;
```
Comment (daskygit) [inline keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c:314]:
```suggestion
```

File: keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+ 
+      case SMILEY:
+      if (record->event.pressed)
+      {
+         SEND_STRING("1s1" SS_TAP(X_DOWN) SS_TAP(X_ENTER));
+      }
+      return false;
+      break;
```
Comment (daskygit) [inline keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c:322]:
```suggestion
```

File: keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+ 
+      case CANBRACKET:
+      if (record->event.pressed)
+      {
+        SEND_STRING("[Can]");
+      }
+      return false;
+      break;
```
Comment (daskygit) [inline keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c:330]:
```suggestion
```

File: keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+ 
+      case REQUESTACCESS:
+      if (record->event.pressed)
+      {
+        SEND_STRING("Hi, I'm a Data Analytics CE for SEA and think access to this document would be useful for me.");
+      }
+      return false;
+      break;
```
Comment (daskygit) [inline keyboards/splitkb/kyria/keymaps/cankoklu/keymap.c:338]:
```suggestion
```

**Review by drashna — APPROVED** [review]

**Review by EricGebhart — COMMENTED** [review]
Looks good except for the changes that Dasky requested are still pending.

**Pull request closed without merging**